### PR TITLE
Remove hardcoded dict types and versions

### PR DESCRIPTION
**Pull request by tommydeboer**
Makes the retrieval and usage of dictionary types and versions generic, so no code changes are needed when a new dictionary (version) is introduced

### CODE REVIEW

File: R/login.R [hunk 1]
```diff
@@ -10,8 +10,7 @@ ds_upload.globals <- new.env()
 #' \dontrun{
 #'
 #' login_data <- data.frame(
-#'   server = "https://armadillo.dev.molgenis.org",
-#'   storage = "https://armadillo-minio.dev.molgenis.org",
+#'   server = "https://armadillo.dev.molgenis.org"
```
Comment (erikzwart) [inline R/login.R:13]: storage is removed, this is compatible with armadillo 2?

**Review by erikzwart — CHANGES_REQUESTED** [review]
Is R/login.R with your changes compatible with armadillo 2?

**Comment by tommydeboer** [comment]
Still needs to be tested on a live Armadillo 3.  Please wait until after the workshops end of January before merging this PR!

File: DESCRIPTION [hunk 2]
```diff
@@ -48,6 +48,7 @@ Imports:
   dplyr (>= 1.0.7),
   opalr (>= 3.1.1),
   MolgenisArmadillo (>= 1.1.3),
+  MolgenisArmadillo (< 2.0.0),
```
Comment (tommydeboer) [inline DESCRIPTION:51]: @erikzwart @marikaris When doing the major dsUpload release, make sure to replace these two lines with:

```MolgenisArmadillo (>= 2.0.0),```

**Review by erikzwart — CHANGES_REQUESTED** [review]
Used the [demo](https://github.com/lifecycle-project/ds-upload/blob/master/inst/examples/data/demo/armadillo/02_armadillo_login-upload.r) set to test, got the following error:
image

Zie session info (dsUpload 4.7.0):
image

**Comment by erikzwart** [comment]
[dsUpload generiek maken dictionaries](https://trello.com/c/opzCGXZo/989-dsupload-generiek-maken-dictionaries)

**Comment by tommydeboer** [comment]
Replaced by #117

**Comment by erikzwart** [comment]
Closing: replaced by #117

**Pull request closed without merging**